src/main/java/thePackmaster/powers/darksoulspack/RemedyPower.java

```diff
@@ -23,9 +25,13 @@ public RemedyPower(AbstractCreature owner, int amount) {
 
     @Override
     public boolean onReceivePower(AbstractPower power, AbstractCreature target, AbstractCreature source) {
-        if (power.type == PowerType.DEBUFF && (amount > 0)){
+        if (power.type == PowerType.DEBUFF && (this.amount > 0)){
             flashWithoutSound();
-            Wiz.att(new ReducePowerAction(target, source, power.ID, 2));
+            if (power.amount > 0)
+                Wiz.att(new ReducePowerAction(target, source, power.ID, 2));
+            else if (power.amount < 0) {
+                Wiz.att(new ReducePowerAction(target, source, power.ID, max(power.amount, -2)));
```

Comment:
This is still incorrect. I'll need to test this code myself, I'll keep the PR open for now until i get around to it. The other stuff looks good though.

Reply:
ok, thanks :)

Reply:
you're right in that it doesn't work properly, going below -2 str/dex/focus breaks it. not quite sure why though

Reply:
is power.amount a positive number when it comes to negative str/dex/focus? if so i can probably fix this

Reply:
got it, its to do with the ReducePowerAction calling AbstractPower.ReducePower():

```public void reducePower(int reduceAmount) {
        if (this.amount - reduceAmount <= 0) {
            this.fontScale = 8.0F;
            this.amount = 0;
        } else {
            this.fontScale = 8.0F;
            this.amount -= reduceAmount;
```

So need to get around this somehow.

Reply:
it seems that the gems pack has come across this problem as well, so I reckon I could take something from there (specifically the gemspack.ReduceDebuffsAction)

Reply:
I gave it a go but couldn't seem to figure it out :(

Reply:
i have updated the code and now it breaks only if the resultant power amount is 0 after the debuff. not sure why.

Reply:
i believe i fixed it, lmk if you spot any more problems